pkg/eib/eib.go

```diff
@@ -107,6 +108,14 @@ func appendElementalRPMs(ctx *image.Context) {
 	}, combustion.ElementalPackages...)
 }
 
+func appendFipsRPMs(ctx *image.Context) {
+	log.AuditInfo("FIPS mode is configured. The necessary RPM packages will be downloaded.")
+	fips := ctx.ImageDefinition.OperatingSystem.Fips
+	if fips {
+		appendRPMs(ctx, nil, "patterns-base-fips")
```

Comment:
Can we extract the name of the RPM as a constant in the combustion package similarly to Elemental ones?

Reply:
Yep, done. I've left it as a list, just incase we need to add to it in the future.